### DISCUSSION TITLE
[Ha ha funny] Is this rounded enough?

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#2224] [Polls]
![image](https://github.com/YimMenu/YimMenu/assets/76223390/6f479213-54d1-4ab0-a151-66d6416c3e23)

**Comment by lonelybud** [comment]
This is something that you can test yourself. Not to open a discussion for such.

**Reply by pierrelasse** [reply]
Since there is no yimmenu discord, i thought i post it here....
That's why there is "Ha ha funny" in the title because there is no help needed.
